src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -644,6 +666,11 @@ namespace winrt::TerminalApp::implementation
     // - LaunchMode enum that indicates the launch mode
     LaunchMode TerminalWindow::GetLaunchMode()
     {
+        if (_contentBounds)
+        {
+            return LaunchMode::DefaultMode;

```

Comment:
I'm sure you're tracking "drag from maximized => make maximized" right?

Reply:
Nevermind, it's not like Edge does it. You can't drag one of those bad boys out and be both maximized and dragging :D